src/routes/docs/products/functions/develop-locally/+page.markdoc

```diff
@@ -0,0 +1,95 @@
+---
+layout: article
+title: Develop locally
+description: Learn to develop Appwrite functions locally.
+---
+
+Develop your Appwrite functions locally to make code changes without redeploying your function on every code change and hot reload your code for faster testing.
+
+# Setup {% #setup %}
+
+1. Install the [Docker CLI](https://www.docker.com/products/docker-desktop/)
+2. Ensure Docker is running in the background
+3. Install the [Appwrite CLI](/docs/tooling/command-line/installation#installation)
+4. [Log in](/docs/tooling/command-line/installation#get-started) to your Appwrite account using `appwrite login`
+5. [Initialize your project](/docs/tooling/command-line/installation#initialization) and create the `appwrite.json` file
+
+We use Docker to replicate the production environment for the local deployment of functions. These can be executed locally with the `appwrite run functions` CLI command, which requires initializing a project with an `appwrite.json` file. The CLI also supports various other [CLI commands](/docs/tooling/command-line/commands).
+
+{% arrow_link href="/docs/tooling/command-line/functions" %}
+Learn more about deploying functions using CLI
+{% /arrow_link %}
+
+# Develop {% #develop %}
+
+Use the `appwrite run functions` command to develop your function locally.
+
+```sh
+appwrite run functions
+? Which function would you like to develop locally? My Awesome Function ("<FUNCTION_ID>")
+
+runtime   │ entrypoint  │ path                           │ commands     
+───────────┼──────────────┼───────────────────────────────────┼─────────────
+node-16.0 │ src/main.js │ functions/<FUNCTION_ID>        │ npm install
```

Comment:
The formatting here doesn't seem right. The lines are supposed to match up so that it looks like a table like:

<img width="478" alt="image" src="https://github.com/user-attachments/assets/f068ffdd-4e31-4ed3-b3a0-90c13896b4e9">